valkey_glide_cluster.stub.php

```
@@ -458,9 +458,36 @@ public function clearConnectionPassword(bool $immediateAuth = false): string;
      * @see ValkeyGlide::config()
-     */
-   //TODO public function config(mixed $route, string $subcommand, mixed ...$extra_args): mixed;
+     *
+     * @example
+     * $cluster->config('randomNode', 'GET', 'timeout');
+     * $cluster->config('allPrimaries', 'GET', ['timeout', 'maxmemory']);
+     * $cluster->config('allNodes', 'SET', 'timeout', '30');
+     * $cluster->config('allNodes', 'SET', ['timeout' => 30, 'loglevel' => 'warning']);
+     * $cluster->config('allPrimaries', 'RESETSTAT');
+     */
+    public function config(mixed $route, string $operation, array|string|null $key_or_settings = null, ?string $value = null): mixed;
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$route'. (undefined)

(UnusedFormalParameter)